index.html

```diff
@@ -18,6 +18,9 @@
                     <li id="no-results">
                         No results
                     </li>
+                    <li>
+                        sadsadasdasdsd
+                    </li>
```

Comment:
Can you please add an id to the `<li>` element? And I made a suggestion to change the content of the list item, you can choose "commit suggestion" to apply my suggestion
Then pull the changes locally by running git pull origin feature/search-2
then add an id, commit and push changes to origin feature/search-2
check if they are visible in this PR
if yes, that's it
```suggestion
                    <li>
                        Frontend developer
                    </li>
```